sklearn/ensemble/_hist_gradient_boosting/_bitset.pxd

```diff
@@ -0,0 +1,13 @@
+# cython: language_level=3
+from .common cimport X_BINNED_DTYPE_C
+from .common cimport BITSET_DTYPE_C
+from .common cimport BITSET_INNER_DTYPE_C
+from .common cimport X_DTYPE_C
+
+cdef void init_bitset(BITSET_DTYPE_C bitset) nogil
+
+cdef void set_bitset(BITSET_DTYPE_C bitset, X_BINNED_DTYPE_C val) nogil
+
+cdef unsigned char in_bitset(BITSET_DTYPE_C bitset, X_BINNED_DTYPE_C val) nogil
+
+cdef unsigned char in_bitset_mv(const BITSET_INNER_DTYPE_C[:] bitset, X_BINNED_DTYPE_C val) nogil
```

Comment:
Should one of these be removed?
Also what does `mv` stand for?

Reply:
"mv" == "memoryview" 

One of the functions works with memoryviews and one works directly with the bitset. The memoryview version is used in `_predictor.pyx` so it would not need to cast to a `BITSET_DTYPE_C`.

Reply:
Rename the function to `in_bitset_memoryview` to be explicit